packages/vue2-jest/lib/process.js

```diff
@@ -11,8 +11,9 @@ const stripInlineSourceMap = require('./utils').stripInlineSourceMap
 const getCustomTransformer = require('./utils').getCustomTransformer
 const loadSrc = require('./utils').loadSrc
 const babelTransformer = require('babel-jest').default
-const compilerUtils = require('@vue/component-compiler-utils')
+const { parse, compileTemplate, compileScript } = require('@vue/compiler-sfc')
```

Comment:
`@vue/compiler-sfc` isn't listed as a dependency of this package, so this import is unsound.

Reply:
Isn't  `@vue/compiler-sfc` Vue 3 only? I'm really surprised this works at all.

Reply:
Also looks like tests are failing - you may need to update a snapshot.

Reply:
Oh I see, they now have a v2.7 for this module: https://www.npmjs.com/package/@vue/compiler-sfc